stylus/settings/palette.styl

```diff
@@ -135,7 +135,6 @@ html, .CozyTheme--normal
     --primaryColor #297EF2
     --primaryColorDark #0B61D6
     --primaryColorLight #5C9DF5
-    --primaryColorLighter #4B93F7 // should not be used
```

Comment:
You just use it here https://github.com/cozy/cozy-ui/pull/1743/commits/a67a446da44a756b03fe8827679004d07ea24e2f#diff-17e9585c61eae7bb5de203c99c5467520ba0d6f1a65cd45e3557287c70cce344R56